### PR TITLE
Custom theme PRA-57

### PR DESCRIPTION
**Pull request by ElenaGor8**
## One Line Description
Added colors and fonts to Chakra default theme

## Requirements

_What this code should do_
extend Chakra default theme

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
     "@chakra-ui/react": "^2.8.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@fontsource/open-sans": "^5.0.7",
+    "@fontsource/raleway": "^5.0.7",
```
Comment (rebekahcallkacz) [inline package.json:12]: NON-BLOCKING: Are we actually using these fonts?

File: src/CustomTheme.js
Comment (rebekahcallkacz) [inline src/CustomTheme.js:1]: BLOCKING: This file should be converted to TypeScript - `CustomTheme.ts`

File: Fonts/Poppins/OFL.txt
Comment (rebekahcallkacz) [inline Fonts/Poppins/OFL.txt:1]: BLOCKING: You shouldn't need this folder `Fonts` or anything in it. The fonts should load in correctly with just the info you included in `CustomTheme.js`

File: package.json (hunk 1 above)
Comment (rebekahcallkacz) [reply package.json:12]: I don't see them being used anywhere and the new fonts show up without me installing these.

File: src/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import ReactDOM from "react-dom/client";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import { ChakraProvider } from "@chakra-ui/react";
+import { ChakraProvider, extendTheme } from "@chakra-ui/react";
```
Comment (rebekahcallkacz) [inline src/index.tsx:3]: BLOCKING: You do not need to import `extendTheme` here

**Review by rebekahcallkacz — CHANGES_REQUESTED** [review]
Congrats on your first PR 🎉 

I see the expected changes and they look great! Most of my comments have to do with code you included that you don't actually need (and thus should remove).

**Comment by ElenaGor8** [comment]
made changes as per Rebecca's requests

**Review by AnnaPestova1 — APPROVED** [review]

**Review by rebekahcallkacz — APPROVED** [review]
This looks great! 

I just want to point out (to make sure it's clear, not because it's an issue), that you've only redefined `gray.900` and `gray.50`. If any of the other grays are used, they are going to be the Chakra defaults.

**Comment by ElenaGor8** [comment]
I have our gray defined as  brandGray: "#f4f4eb"
900 to 50 I should probably delete, as they are from default theme, right?

**Comment by rebekahcallkacz** [comment]
> I have our gray defined as brandGray: "#f4f4eb" 900 to 50 I should probably delete, as they are from default theme, right?

If you're not using them, then, yes, I'd remove them!

**Pull request merged**